### PR TITLE
Adds new abductee objectives and replaces some lame ones.

### PR DESCRIPTION
**Pull request by harmonyn**
The latter primarily means "make solitary objectives more RP-oriented."

### CODE REVIEW

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 1]
```diff
@@ -396,16 +401,19 @@
 	explanation_text = "Start a collection of corpses. Don't kill people to get these corpses."
 
 /datum/objective/abductee/floors
-	explanation_text = "Replace all the floor tiles with carpeting, wooden boards, or grass."
+	explanation_text = "Replace all the floor tiles with carpeting, grass or bling."
```
Comment (PKPenguin321) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Keep wood boards and also respect the oxford comma

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 2]
```diff
@@ -396,16 +401,19 @@
...
-	explanation_text = "Ensure the station is in absolutely pristine condition."
+	explanation_text = "The CEO of Nanotrasen is coming! Ensure the station is in absolutely pristine condition."
+
+/datum/objective/abductee/nowalls
+	explanation_text = "The crew must get to know one another better. Break down the walls inside the station!"
 
-/datum/objective/abductee/window
-	explanation_text = "Replace all normal windows with reinforced windows."
+/datum/objective/abductee/wall
+	explanation_text = "There are too many new crew members coming to this station. You must build a wall!"
```
Comment (PKPenguin321) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: I feel like this will just lead to them walling off arrivals which seems sorta griefy

Comment (Haevacht) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Why the dumb refrence.

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 3]
```diff
@@ -459,16 +464,30 @@
...
+
 /datum/objective/abductee/music
 	explanation_text = "Start playing music, you're the best musician ever. If anyone hates it, beat them on the head with your instrument!"
-	
+
+/datum/objective/abductee/clown
+	explanation_text = "The clown is not funny. You can do better! Make the crew laugh!"
+
+/datum/objective/abductee/party
+	explanation_text = "You're throwing a huge rager. The whole crew needs to come... OR ELSE!"
```
Comment (PKPenguin321) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: ERP is banned homie

Comment (bgobandit) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: what

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 4]
```diff
@@ -354,9 +354,14 @@
...
 
 /datum/objective/abductee/steal/New()
-	var/target = pick(list("pets","lights","monkeys","fruits","shoes","bars of soap"))
+	var/target = pick(list("pets","lights","monkeys","fruits","shoes","bars of soap", "weapons", "computers", "organs"))
 	explanation_text+=" [target]."
 
+/datum/objective/abductee/paint
+	explanation_text = "The station is hideous. You must color it all"
+	var/color = pick(list("red", "blue", "green", "yellow", "orange", "purple", "black", "in rainbows"))
+	explanation_text+= "[color]!"
```
Comment (Haevacht) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Probably just do the colour picking before the line and have the explanation text all in one line. Considering it's fucking with your compile.

Comment (bgobandit) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: yeah

File: code/game/gamemodes/miniantags/abduction/abduction.dm (hunk 3 above)
Comment (PKPenguin321) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Is joke

File: code/game/gamemodes/miniantags/abduction/abduction.dm (hunk 2 above)
Comment (bgobandit) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: I mean they are after all quasi-antags

**Comment by Haevacht** [comment]
Other options for speech patters are like haiku, limerick, without vowels, never using certain letter, with a lisp. A manually added lisp not a lizard lisp.

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 5]
```diff
@@ -459,16 +467,30 @@
...
 /datum/objective/abductee/licky
 	explanation_text = "You must lick anything that you find interesting."
-	
+
 /datum/objective/abductee/music
 	explanation_text = "Start playing music, you're the best musician ever. If anyone hates it, beat them on the head with your instrument!"
-	
+
+/datum/objective/abductee/clown
+	explanation_text = "The clown is not funny. You can do better! Make the crew laugh!"
```
Comment (RandomMarine) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: What if you are the clown?

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 6]
```diff
@@ -354,9 +354,23 @@
...
-	var/target = pick(list("pets","lights","monkeys","fruits","shoes","bars of soap"))
+	var/target = pick(list("pets","lights","monkeys","fruits","shoes","bars of soap", "weapons", "computers", "organs"))
 	explanation_text+=" [target]."
 
+/datum/objective/abductee/paint
+	explanation_text = "The station is hideous. You must color it all"
+
+/datum/objective/abductee/paint/New()
+	var/color = pick(list("red", "blue", "green", "yellow", "orange", "purple", "black", "in rainbows", "in blood"))
+	explanation_text+= "[color]!"
```
Comment (MrPerson) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: this should be " [color]!" or else you'll get "The station is hideous. You must color it allblack!"

File: code/game/gamemodes/miniantags/abduction/abduction.dm (hunk 5 above)
Comment (MrPerson) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: git funny

Comment (bgobandit) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: the thought of the clown being brainwashed into outdoing himself is hilarious imo

File: code/game/gamemodes/miniantags/abduction/abduction.dm (hunk 6 above)
Comment (bgobandit) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: oops

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 7]
```diff
@@ -466,19 +466,13 @@
...
-	explanation_text = "You must lick anything that you find interesting."
+	explanation_text = "Go have a good conversation with the singularity/tesla/supermatter crystal. Bonus points if it responds."
 
 /datum/objective/abductee/music
-	explanation_text = "Start playing music, you're the best musician ever. If anyone hates it, beat them on the head with your instrument!"
+	explanation_text = "You burn with passion for music. Share your vision. If anyone hates it, beat them on the head with your instrument!"
 
 /datum/objective/abductee/clown
-	explanation_text = "The clown is not funny. You can do better! Make the crew laugh!"
+	explanation_text = "The clown is not funny. You can do better! Steal his audience and make the crew laugh!"
```
Comment (RandomMarine) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Did you just assume the clown's gender??!?!??!?!???!??1?

Comment (bgobandit) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: women aren't funny

Comment (Haevacht) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Dumb jokes aside, use [p_their()] instead of his.

Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: You can't because multiple clowns just use their
Maybe rephrase it to `Clowns are not funny. You can do better! Steal their audience and make the crew laugh!`

Comment (Haevacht) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: My bad.

Comment (bgobandit) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: this is bikeshedding

the clown... he would not approve

Comment (optimumtact) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Are you arguing with a maintainer?

Comment (bgobandit) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: yes

ohhhhh

Comment (optimumtact) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: A - always
B - be 
C - closing

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 8]
```diff
@@ -443,7 +451,7 @@
 	explanation_text = "Nanotrasen is abusing the animals! Save as many as you can!"
 
 /datum/objective/abductee/defect
-	explanation_text = "Defect from your employer."
+	explanation_text = "Fuck the system! Defect from the station and start an independent colony in space, Lavaland or the derelict. Recruit crewmates if you can."
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Christian Server!
